scipy/linalg/misc.py

```diff
@@ -142,14 +147,14 @@ def norm(a, ord=None, axis=None, keepdims=False, check_finite=True):
         a = np.asarray(a)
 
     # Only use optimized norms if axis and keepdims are not specified.
```

Comment:
This comment is now even more out of date--it doesn't mention the dtype or the size.  Let's just remove it--it doesn't add any information that isn't clear from the condition in the `if`-statement.
```suggestion
```